activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ModelControllerIT.java

```diff
@@ -189,19 +187,20 @@ public void should_returnStatusCreatedAndProcessModelExtensions_when_creatingPro
                                         .add("myIntegerConstant",
                                              10)
                                         .build());
-        ModelEntity processModel = processModelWithExtensions("process-model-extensions",
-                                                              extensions);
+        Map<String, Extensions> extensionsMap = new HashMap<String, Extensions>();
```

Comment:
extensionsMap, maybe just extensions, or processesToExtensions to be very specific